dataland-frontend/src/components/pages/ViewFrameworkData.vue

```diff
@@ -1,29 +1,34 @@
 <template>
-  <ViewSingleDatasetDisplayBase
-    v-if="singleViewFrameworks.includes(dataType)"
-    :dataType="dataType"
-    :companyId="companyId"
-    :dataId="dataId"
-    :reportingPeriod="reportingPeriod"
-  />
-  <ViewMultipleDatasetsDisplayBase
-    v-if="multiViewFrameworks.includes(dataType)"
-    :dataType="dataType"
-    :companyId="companyId"
-    :dataId="dataId"
-    :reportingPeriod="reportingPeriod"
-  />
+  <AuthenticationWrapper :disable-authentication-wrapper="viewInPreviewMode">
```

Comment:
Reminder for myself: Think about this, if their is a possibility to misuse this deactivation of the auth wrapper

Reply:
There is no possibility to misuse this, since this is only a "visual" element in the Frontend.
Of course someone could go to the dev tools and activate the wrapper somehow, but that person could still not fetch any sensitive data without a valid keycloak token.
So no risk

Reply:
ok